modules/op-release-notes-1-15.adoc

```diff
@@ -0,0 +1,430 @@
+** `GRPC_WORKER_POOL`: The size of the GRPC worker pool
+** `K8S_QPS`: The Kubernetes client QPS setting
+** `K8S_BURST`: The Kubernetes client burst QPS setting
++
+If you want to use this setting, when configuring {tekton-results} you must also use alternate specs for several other configuration parameters, as listed in the following table. Both the regular and the alternate parameter specs are in the `TektonResult` CR.
++
+.Alternate configuration parameters for {tekton-results}
+[options="header"]
+|===
+
+| Regular parameter spec | Alternate parameter spec
+
+| `logs_api` | `options.configMaps.tekton-results-api-config.data.config.LOGS_API`
+| `log_level` | `options.configMaps.tekton-results-api-config.data.config.LOG_LEVEL`
+| `db_port` | `options.configMaps.tekton-results-api-config.data.config.DB_PORT`
+| `db_host` | `options.configMaps.tekton-results-api-config.data.config.DB_HOST`
+| `logs_path` | `options.configMaps.tekton-results-api-config.data.config.LOGS_PATH`
+| `logs_type` | `options.configMaps.tekton-results-api-config.data.config.LOGS_TYPE`
+| `logs_buffer_size` | `options.configMaps.tekton-results-api-config.data.config.LOGS_BUFFER_SIZE`
+| `auth_disable` | `options.configMaps.tekton-results-api-config.data.config.AUTH_DISABLE`
+| `db_enable_auto_migration` | `options.configMaps.tekton-results-api-config.data.config.DB_ENABLE_AUTO_MIGRATION`
+| `server_port` | `options.configMaps.tekton-results-api-config.data.config.SERVER_PORT`
+| `prometheus_port` | `options.configMaps.tekton-results-api-config.data.config.PROMETHEUS_PORT`
+| `gcs_bucket_name` | `options.configMaps.tekton-results-api-config.data.config.GCS_BUCKET_NAME`
+
+|===
++
+For the configuration parameters not listed in this table, use the regular specs as described in the documentation.
++
+[IMPORTANT]
+====
+Use the alternate parameter specs only if you need to use the additional settings in the `options.configMaps.tekton-results-api-config.data.config` section of the `TektonResult` CR.
+====
+
+* With this update, you can use the {tekton-results} API to retrieve the Go profiling data for {tekton-results}.
+
+* Before this update, {tekton-results} checked the user authentication when displaying every fragment of log data. With this update, {tekton-results} checked the user authentication only once per log data request. This change improves the response time for the {tekton-results} log API, which is used for displaying logs using the command line utility.
+
+[id="breaking-changes-1-15_{context}"]
+== Breaking changes
+
+* Before this update, you could specify the `git-provider`, `sender`, and `branch` labels in a {pac} pipeline run YAML file. With this update, {pac} no longer supports these labels, Instead, you can specify `pipelinesascode.tekton.dev/git-provider`, `pipelinesascode.tekton.dev/sender`, and `pipelinesascode.tekton.dev/branch` as annotations.
```

Comment:
PAC use to set those labels and end user use to rely on those labels 

```suggestion
* Before this update, {pac} used to specify the `git-provider`, `sender`, and `branch` labels in a {pac} pipeline run YAML file. With this update, {pac} no longer supports these labels, Instead, {pac} now specify `pipelinesascode.tekton.dev/git-provider`, `pipelinesascode.tekton.dev/sender`, and `pipelinesascode.tekton.dev/branch` as annotations.
```
How about this ??

Reply:
I'm really not sure what is going on here - are you saying this is somethingpurely internal to PAC, not something the user could set?

Reply:
fixed as best I can, though. Removed "YAML file" as PAC clearly does not edit files in Git?

Reply:
LGTM